client/web/src/enterprise/insights/components/form/monaco-field/MonacoField.module.scss

```
@@ -32,27 +42,60 @@
             box-shadow: 0 0 0 2px var(--danger-3);
         }
     }
+}
+
+.monaco-field {
+    position: relative;
+    display: flex;
+    width: 100%;
+    min-width: 0;
+    height: auto;
 
     :global(.scroll-decoration) {
         display: none;
     }
 
+    &:global(.with-invalid-icon) {
+        &:global(.is-invalid)::before,
+        :global(.is-valid)::before {
+            // Bootstrap styles for form control with validation icon
+            padding-right: calc(1.4285714286em + 0.75rem);
```

```suggestion
            padding-right: calc(1.4285714286rem + 0.75rem);
```

Stylelint is real mad about this use or `em`

---

Oh, actually we don't need these padding styles anymore. I will remove this.